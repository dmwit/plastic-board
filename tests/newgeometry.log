*geometry* verbose mode - [ newgeometry ] result:
* driver: pdftex
* paper: <default>
* layout: <same size as paper>
* layoutoffset:(h,v)=(0.0pt,0.0pt)
* modes: 
* h-part:(L,W,R)=(125.19194pt, 363.91112pt, 125.19194pt)
* v-part:(T,H,B)=(95.39737pt, 556.47656pt, 143.09605pt)
* \paperwidth=614.295pt
* \paperheight=794.96999pt
* \textwidth=363.91112pt
* \textheight=556.47656pt
* \oddsidemargin=52.92195pt
* \evensidemargin=52.92195pt
* \topmargin=-13.87262pt
* \headheight=12.0pt
* \headsep=25.0pt
* \topskip=11.0pt
* \footskip=30.0pt
* \marginparwidth=4.0pt
* \marginparsep=10.0pt
* \columnsep=28.45274pt
* \skip\footins=10.0pt plus 4.0pt minus 2.0pt
* \hoffset=0.0pt
* \voffset=0.0pt
* \mag=1000
* \@twocolumntrue
* \@twosidefalse
* \@mparswitchfalse
* \@reversemarginfalse
* (1in=72.27pt=25.4mm, 1cm=28.453pt)
