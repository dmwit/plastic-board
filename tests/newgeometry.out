([],((Nothing,Nothing),Boring "*geometry* verbose mode - [ newgeometry ] result:"))
([],((Nothing,Nothing),Boring "* driver: pdftex"))
([],((Nothing,Nothing),Boring "* paper: <default>"))
([],((Nothing,Nothing),Boring "* layout: <same size as paper>"))
([],((Nothing,Nothing),Boring "* layoutoffset:(h,v)=(0.0pt,0.0pt)"))
([],((Nothing,Nothing),Boring "* modes: "))
([],((Nothing,Nothing),Boring "* h-part:(L,W,R)=(125.19194pt, 363.91112pt, 125.19194pt)"))
([],((Nothing,Nothing),Boring "* v-part:(T,H,B)=(95.39737pt, 556.47656pt, 143.09605pt)"))
([],((Nothing,Nothing),Boring "* \\paperwidth=614.295pt"))
([],((Nothing,Nothing),Boring "* \\paperheight=794.96999pt"))
([],((Nothing,Nothing),Boring "* \\textwidth=363.91112pt"))
([],((Nothing,Nothing),Boring "* \\textheight=556.47656pt"))
([],((Nothing,Nothing),Boring "* \\oddsidemargin=52.92195pt"))
([],((Nothing,Nothing),Boring "* \\evensidemargin=52.92195pt"))
([],((Nothing,Nothing),Boring "* \\topmargin=-13.87262pt"))
([],((Nothing,Nothing),Boring "* \\headheight=12.0pt"))
([],((Nothing,Nothing),Boring "* \\headsep=25.0pt"))
([],((Nothing,Nothing),Boring "* \\topskip=11.0pt"))
([],((Nothing,Nothing),Boring "* \\footskip=30.0pt"))
([],((Nothing,Nothing),Boring "* \\marginparwidth=4.0pt"))
([],((Nothing,Nothing),Boring "* \\marginparsep=10.0pt"))
([],((Nothing,Nothing),Boring "* \\columnsep=28.45274pt"))
([],((Nothing,Nothing),Boring "* \\skip\\footins=10.0pt plus 4.0pt minus 2.0pt"))
([],((Nothing,Nothing),Boring "* \\hoffset=0.0pt"))
([],((Nothing,Nothing),Boring "* \\voffset=0.0pt"))
([],((Nothing,Nothing),Boring "* \\mag=1000"))
([],((Nothing,Nothing),Boring "* \\@twocolumntrue"))
([],((Nothing,Nothing),Boring "* \\@twosidefalse"))
([],((Nothing,Nothing),Boring "* \\@mparswitchfalse"))
([],((Nothing,Nothing),Boring "* \\@reversemarginfalse"))
([],((Nothing,Nothing),Boring "* (1in=72.27pt=25.4mm, 1cm=28.453pt)"))
